website/src/content/api-reference/commercetools-frontend-i18n.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "@/i18n"
+---
+<Subtitle>
+
+The internationalization messages from the various application-kit packages are contained as JSON data in this package.
+
+</Subtitle>
+
+# Supported locales
+
+- English               `en`
+- German                `de`
+- Spanish               `es`
+- French (France)       `fr-FR`
+- Simplified Chinese    `zh-CN`
+
+# Installation
+
+**Using npm**
+
+```sh
+npm install --save @commercetools/i18n
+```
+**Using yarn**
+
+```sh
+yarn add @commercetools/i18n
+```
+
+# Usage
```

Comment:
Usually I would like to focus on the page structure first. It's important to define it as it helps identifying the flow of the content and as a consequence where to put what info.

Let's maybe have a sync about it?